packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -192,6 +193,23 @@ class PoolMaster extends Component {
   }
 }
 
+@connectStore(() => ({
+  defaultSr: createGetObject((_, { pool }) => pool.default_SR),
+}))
+class SelectDefaultSr extends Component {
+  _onChange = sr => setDefaultSr(sr)
+
+  render() {
+    const { defaultSr } = this.props
+
+    return (
+      <XoSelect onChange={this._onChange} value={defaultSr.id} xoType='SR'>
+        {defaultSr.name_label}
```

Comment:
Use the `Sr` component to render the SR and handle the case where there is no default SR set on the pool.